osschallenge/models.py

```diff
@@ -34,10 +35,18 @@ class Profile(models.Model):
     contact = models.CharField(max_length=50)
     key = models.CharField(max_length=10, unique=True)
     picture = ThumbnailerImageField(upload_to='profile-pictures', null=True)
+    rank = models.ForeignKey(Rank, default=2)
```

Comment:
Why is the default 2? Wouldn't that mean that a contributer is automatically on rank 2 (padawan) instead of rank 1 (youngling)?

Reply:
rank 1 required_points = 0, means on first login you would get an levelup redirect and set rank to 2 immediately
rank 2 required_points = 15, no levelup on first login and still works

Reply:
Ah I see. So the rank is still youngling until the contributor reaches 15 points?

Reply:
Yes, exactly.

Reply:
:+1: